Servers/models/ISO-42001/annexCategoryISO.model.ts

```
@@ -0,0 +1,111 @@
+  })
+  justification_for_exclusion?: string;
+
+  @Column({
+    type: DataType.STRING
+  })
+  implementation_description?: string;
+
+  @Column({
+    type: DataType.JSONB
+  })
+  evidence_links?: Object[];
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)